exec/java-exec/src/test/java/org/apache/drill/PlanTestBase.java

```diff
@@ -92,7 +91,20 @@ public static void testPlanMatchingPatterns(String query, String[] expectedPatte
 
   public static void testPlanMatchingPatterns(String query, Pattern[] expectedPatterns, Pattern[] excludedPatterns)
     throws Exception {
-    final String plan = getPlanInString("EXPLAIN PLAN for " + QueryTestUtil.normalizeQuery(query), OPTIQ_FORMAT);
+    testPlanMatchingPatterns(query, OPTIQ_FORMAT, expectedPatterns, excludedPatterns);
+  }
+
+  public static void testPlanMatchingPatterns(String query, String planFormat,
```

Comment:
Minor comment  here. `planFormat` was not exposed to tests earlier. Should we restrict the format to `OPTIQ_FORMAT` and `JSON_FORMAT`

Reply:
Done